src/packagedcode/rpm.py

```diff
@@ -231,8 +231,7 @@ class RpmInstalledNdbDatabaseHandler(BaseRpmInstalledDatabaseHandler):
 class RpmInstalledSqliteDatabaseHandler(BaseRpmInstalledDatabaseHandler):
     # used by newer RHEL/CentOS/Fedora
     datasource_id = 'rpm_installed_database_sqlite'
-    path_patterns = ('*var/lib/rpm/rpmdb.sqlite',)
-    filetypes = ('berkeley',)
+    path_patterns = ('*var/lib/rpm/rpmdb.sqlite', '*usr/lib/sysimage/rpm/rpmdb.sqlite',)
```

Comment:
We could also relax it even more like in:
```suggestion
    path_patterns = ('*rpm/rpmdb.sqlite',)
```

Reply:
I like that. I'll make the change. Also. I have a pending change to support rpm-ostree variants too.

Reply:
Thanks!

Reply:
I created an integration test, but I noted that unrelated tests seem to fail (unsure about those). Also, I was not sure about the `--system-package` testing. Happy to author, but would need a pointer to the starting thread.